activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -1049,8 +1049,8 @@ def assume_migrated_upto_version(version, migrations_paths)
         sm_table = quote_table_name(ActiveRecord::SchemaMigration.table_name)
 
         migrated = ActiveRecord::SchemaMigration.all_versions.map(&:to_i)
-        versions = ActiveRecord::Migrator.migration_files(migrations_paths).map do |file|
-          ActiveRecord::Migrator.parse_migration_filename(file).first.to_i
+        versions = migration_context.migration_files(migrations_paths).map do |file|
```

Comment:
Can we eliminate the `migration_paths` parameter here?  I think `migration_context` contains the same information.

Reply:
Doh I missed that one.